Formula/inetutils.rb

```diff
@@ -37,7 +38,7 @@ def install
       args << "--program-prefix=g"
     end
     system "./configure", *args
-    system "make", "install"
+    system "make", "SUIDMODE=", "install"
```

Comment:
What is `SUIDMODE` supposed to do?

Reply:
The generated Makefiles contain the following:
```
SUIDMODE = -o root -m 4755
```
which in turn are passed to GNU install to force setuid install of the missing binaries.

Reply:
Bit weird that `install` just fails silently then.

Reply:
I guess it was intentional. From `src/Makefile`:
```
install-rlogin-hook:
        -@$(INSTALL_PROGRAM) $(SUIDMODE) $(AM_INSTALL_PROGRAM_FLAGS) rlogin $(DESTDIR)$(bindir)/`echo rlogin|sed '$(transform)'` ; OUTCOME=$$?; \
        if test $$OUTCOME -ne 0; then \
          echo "WARNING: Failed to install rlogin (exit code $$OUTCOME)"; \
          echo "NOTE: This program often needs to be owned by root."; \
        fi
```
So upstream actually catches the failed install and prints a diagnostic message (which appears in the Homebrew logs, and helped me quickly zero in on the underlying issue. I guess it's because, at least on Linux, there are OS capabilities that can be set to run these programs without root: https://github.com/guillemj/inetutils/blob/master/README#L59-L67